packages/ecs-agent/ecs.config

```
@@ -1,2 +1,6 @@
 ECS_LOGFILE=/var/log/ecs/ecs-agent.log
 ECS_LOGLEVEL="{{settings.ecs.loglevel}}"
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{#if @last}},{{/if}}{{/each}}169.254.169.254,169.254.170.2,/var/run/docker.sock
```

```suggestion
NO_PROXY={{#each settings.network.no-proxy}}{{this}},{{/each}}169.254.169.254,169.254.170.2,/var/run/docker.sock
```
Wouldn't this wind up being the same?  Can simplify because we know there are values after.